src/components/ui/Button/index.tsx

```diff
@@ -0,0 +1,14 @@
+import React from "react";
+import {
+  Button as RNButton,
+  ButtonProps as RNButtonProps,
+} from "react-native-elements";
+
+interface ButtonInputProps extends RNButtonProps {
```

Comment:
Would also be nice to have a usage where we declare the button type so we don't have to worry about using colours.

E.g. using it like this:

```
<Button type="primary" />
```

Would show a teal button with white text already